sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -47,19 +47,13 @@ public ConfidentialLedgerClient(Uri ledgerUri, TokenCredential credential) : thi
         /// <summary> The constitution is a script that assesses and applies proposals from consortium members. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <remarks>
+        /// Below is the JSON schema for the request and response payloads.
+        /// Additional information can be found in the service REST API documentation:
+        /// https://docs.microsoft.com/rest/api/ConfidentialLedger/ConfidentialLedger/GetConstitution
```

Comment:
I get a 404 for this link.

Reply:
@christothes, would you be able to work with the ConfidentialLedger team to update their swagger file with https://swagger.io/specification/v2/#externalDocumentationObject values that link to REST API docs?